crates/fluvio/src/consumer.rs

```diff
@@ -575,8 +576,19 @@ mod publish_stream {
 /// MAX FETCH BYTES
 static MAX_FETCH_BYTES: Lazy<i32> = Lazy::new(|| {
     use std::env;
+    use fluvio_protocol::Encoder;
+    use crate::dataplane::fetch::FetchResponse;
+    use crate::dataplane::fetch::FetchableTopicResponse;
+    use crate::dataplane::fetch::FetchablePartitionResponse;
+    use crate::dataplane::batch::MemoryRecords;
     let var_value = env::var("FLV_CLIENT_MAX_FETCH_BYTES").unwrap_or_default();
-    let max_bytes: i32 = var_value.parse().unwrap_or(1000000);
+    let max_bytes: i32 = var_value.parse().unwrap_or_else(|_| {
+        FetchResponse::<MemoryRecords>::default().write_size(0) as i32
+            + FetchableTopicResponse::<MemoryRecords>::default().write_size(0) as i32
+            + FetchablePartitionResponse::<MemoryRecords>::default().write_size(0) as i32
+            + 64 // using max size of topic name
```

Comment:
Where is this come from?

Reply:
I am using https://github.com/infinyon/fluvio/blob/5d1ca946e0be25c866068b3d165e3891a3522a6e/crates/fluvio-dataplane-protocol/src/fetch/response.rs#L54 which has a `name` attribute which is a String with the topic name, I thought that topic name size is limited to the kubernetes max size of a label which is 63, not sure if that is the actual limit of a topic name

Reply:
I found that, actually the max length is 252 (so it is 255 in bytes (similar to kafka)) I am adding a const for that and changing that here